README.md

```diff
@@ -1,2 +1,28 @@
-# biovectors
-Detecting Biomedical Relationships using Word2vec On Pubtator Central
+# Biovectors
+Detecting Biomedical Relationships using Word2vec On Pubtator Central Abstracts
+
+## Installation Instructions
+
+Biovectors uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager.
+Before moving on to the instructions below, please make sure to have it installed.
+[Download conda here!!](https://docs.conda.io/en/latest/miniconda.html)
+
+Once everything has been installed, type following command in the terminal:
+
+```bash
+bash install.sh
+```
+_Note_:
+There is a bash command within the install.sh that only works on unix systems.
+If you are on windows (and possibly mac), you should remove that file or execute each command individually.
```

Comment:
Just to clarify you're saying they can remove the bash script? Not a mac person but I thought that their OS was similar to linux that it could run this bash setup but the only issue might be certain libraries not being available. For windows, can they use the anaconda terminal? Its been a while since I used this. I am not as good about specifying how to run this in other OS so this will be something I will need to update as well.

Reply:
>Just to clarify you're saying they can remove the bash script?

No that was a slip of the mind. I was saying they should execute commands individually or remove the *line* that has the source command in it. 

>For windows, can they use the anaconda terminal? Its been a while since I used this. I am not as good about specifying how to run this in other OS so this will be something I will need to update as well.

For windows they don't have a bash terminal built in, which makes it hard to have an install script. @ben-heil enlighten me that windows has a terminal emulator, so windows people can also use my scripts. I don't think anaconda terminal will work as I have never used it :man_shrugging:.